tasks/build.yml

```diff
@@ -37,11 +37,16 @@
     name: "openjdk-{{ sansible_java_version }}-jdk"
     state: present
 
+- name: Get DEB architecture
+  command: dpkg --print-architecture
+  register: deb_architecture
```

Comment:
We namespace all of our vars to avoid collisions with the global namespace, can this be changed to "sansible_java_deb_architecture".